mainnet-contracts/src/CarrotStaker.sol

```diff
@@ -34,7 +34,7 @@ contract CarrotRestaking is ERC20, Ownable, ICarrotRestaking {
     }
 
     /**
-     * @inheritdoc ICarrotRestaking
+     * @inheritdoc ICarrotStaker
      */
     function stake(uint256 amount) external {
         CARROT.transferFrom(msg.sender, address(this), amount);
```

Comment:
You don't use named parameters here, but you use them in the event where there is only one parameter (bool) :D 
It is more important to use them whenever you have multiple values of same type